mnefun/_mnefun.py

```diff
@@ -1353,7 +1353,7 @@ def do_preprocessing_combined(p, subjects):
                 raise NameError('File not found (' + r + ')')
 
         bad_file = op.join(bad_dir, 'bad_ch_' + subj + p.bad_tag)
-        if isinstance(p.auto_bad, float):
+        assert isinstance(p.auto_bad_reject, dict)
```

Comment:
I think you still want the original conditional `if isinstance(p.auto_bad, float)`